rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -1266,6 +1297,21 @@ namespace rsx
 			m_invalidated_memory_range = utils::address_range::start_end(0x2 << 28, constants::local_mem_base + local_mem_size - 1);
 			handle_invalidated_memory_range();
 		}
+		else if (state != FIFO_state::lock_wait && new_get_put != umax)
```

Comment:
Swap the checks when rebasing. state is almost never lock-wait.

Reply:
Done.